lib/lbt/analyzer/analyzeLibraryJS.js

```diff
@@ -82,6 +82,60 @@ async function analyze(resource) {
 	return libInfo;
 }
 
+const initLibraryDefintions = [];
```

Comment:
This module should not hold state in between calls.
I see potential side-effects when this module is called multiple times as this array is not cleared.

Reply:
I wasn't referring to the two-step approach which is indeed needed. I'm rather questioning why the variable is declared in the global scope of the module rather than within the "analyze" function scope.

Furthermore I don't think it's a good practice to collect that information in the same function that is supposed to tell whether a node is an init call. This should be separated.